### PR TITLE
Ticker

### PR DESCRIPTION
**Pull request by cbessee**
### Description:

Adds a ticker to the bottom of the app which will display a list of currently playing acts for each stage. Ticker is hidden if there are no artistis currently playing

### UI images:

Screen Shot 2019-07-17 at 2 21 53 PM

### Unit Test Approach:

Describe unit tests being added with that PR.

### Test Results:
- [ ] Did you run `npm test` and did the results pass? NA
- [ ] Did you add unit tests that cover your changes? NA

Describe other (non-unit) test results here.

### CODE REVIEW

File: src/components/Ticker/Ticker.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    this.fetchIntervalHandle = null;
+  }
+
+  fetchNowPlaying() {
+    const moment = extendMoment(Moment);
+    const now = moment().tz("America/Los_Angeles");
+    const nowPlaying = schedule.filter((artist) => {
+      const start = moment(artist.start_time, 'YYYY-MM-DD HH:mm');
+      const end   = moment(artist.end_time, 'YYYY-MM-DD HH:mm');
+      const range = moment.range(start, end);
```
Comment (danielduhh) [inline src/components/Ticker/Ticker.js:23]: 💯

**Review by danielduhh — COMMENTED** [review]
Wow, this is awesome! 
UI suggestions:

Can we shrink the size of the "now playing" block to leave more real estate for the artist name and stage? @holly-moore

**Comment by cbessee** [comment]
> Wow, this is awesome!
> UI suggestions:
> 
> Can we shrink the size of the "now playing" block to leave more real estate for the artist name and stage? @holly-moore

Yep! Holly is sitting across from me and taking a look at it now!

**Comment by cbessee** [comment]
@danielduhh Ok - updated now playing so it's a bit smaller based on @holly-moore 's suggestion. Also updated the event list and right drawer tabs to use the CHBP blue color instead of teal. The teal color was causing us to fail accessibility checks horribly.

**Comment by danielduhh** [comment]
Okay that's awesome. Thanks for doing that and thanks @holly-moore ! Curious, are you using lighthouse to measure accessibility usage ?

**Review by danielduhh — APPROVED** [review]

**Pull request merged**